prime-router/src/main/kotlin/FakeReport.kt

```diff
@@ -205,7 +214,7 @@ class FakeDataService : Logging {
             Element.Type.TABLE, Element.Type.TABLE_OR_BLANK -> createFakeTableValue(element)
             Element.Type.HD -> element.default ?: "0.0.0.0.1"
             Element.Type.EI -> element.default ?: "SomeEntityID"
-            Element.Type.ID -> faker.numerify("######")
+            Element.Type.ID -> createFakeValueFromValueSet(element).ifEmpty { faker.numerify("##D#######") }
```

Comment:
Was that D in the ##### on purpose?  Looks like you added some extra # as well.  Just making sure.  I do not know what it should be.